chore: festive support update

joethreepwood opened the pull request:
Adds a support message to the modal. To be merged on 23rd/24th December, and removed on December 28th. 

Over Christmas, someone on the support team will be working as normal on all days except for the 25th, 26th, and 27th. On these days someone in each timezone will check in on tickets. We'll only be responding to critical severity tickets (e.g. those with significant business impact) and escalating them as needed. All other tickets will be backlogged until when we're back.

Here's who's covering when:

|          | 25th   | 26th    | 27th   |
|----------|--------|---------|--------|
| US times | @benHPostHog  | @slshults  | @benHPostHog |
| EU times | @darkopia  | @abigailbramble  | @darkopia   |

We have a 12 hour SLA on responding to these tickets, so support engineers should be able to check in on these tickets with minimal disruption to their day, hopefully!

If there's an incident during this time which requires comms, we'll follow the usual processes. I'll be available if needed, but in a US timezone (Utah).

If there are any changes to this plan, I'll keep this top comment updated.

zlwaterfield committed:
Improve the support message

zlwaterfield commented:
@joethreepwood I've updated this to use a better structure that is easier to change and turn on/off. It now has two variables 
 for the title and description where you can change the content and then turn it on with a feature flag. 

If we wanted to take this a step further, we could set the content inside the feature flag so no code changes are needed, but that may be overkill.

joethreepwood commented:
@abigailbramble handing this over to you - should be good to go, but: with great power comes tedious responsibilities!

joethreepwood commented:
This isn't stale, fyi.

abigailbramble committed:
Update SidePanelSupport.tsx

Update to support message: https://posthog.slack.com/archives/C075D3C5HST/p1734429690023219

abigailbramble committed:
Update constants.tsx

abigailbramble committed:
Merge branch 'master' into christmas-support-2024
